src/xenialdan/Backpack/Loader.php

```diff
@@ -219,6 +224,19 @@ public function onEnable()
 	{
 		@mkdir($this->getDataFolder() . "players");
 		$this->getServer()->getPluginManager()->registerEvents(new EventListener(), $this);
+
+		if (!PacketHooker::isRegistered()) {
```

Comment:
Still dont understand why plugins MUST check this.. like for real, nobody gives a f on this exception
register could just return false or sth when already registered

Reply:
yeah it's more of an issue when phars are packaged, which is when it's almost always false (exception being if you /reload server)